Mapping/src/test/scala/nl/lumc/sasc/biopet/test/mapping/MappingSingle.scala

```diff
@@ -29,6 +31,14 @@ trait MappingSingle extends MappingSuccess with TestReference {
                "bases" :: "num_total" :: Nil,
                _ shouldBe 1000000,
                skipFlexiprep != Some(true))
+
+  override def configs = {
+    val programs: List[String] = CollectMultipleMetrics.Programs.values.toList.collect({
```

Comment:
Been thinking about this, we should remove this and fix this inside biopet, this is just a workaround for something we can fix ourselfs